controllers/api/v1alpha1/cfprocess_webhook_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("CFProcessMutatingWebhook", func() {
 		cfProcess = &korifiv1alpha1.CFProcess{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      cfProcessGUID,
-				Namespace: namespace,
+				Namespace: rootNamespace,
```

Comment:
CFProcesses won't exist in the root namespace in practice, so I'd undo this change.

Reply:
As above